package.json

```diff
@@ -43,7 +43,7 @@
     "path": "^0.12.7",
```

Comment:
Unrelated, but this certainly shouldn't be in devDependencies since we're probably wanting the native Node `path` module.
I'll take a peek at the upstream shield template, but we should ✂️ this out while we're here.